db/schema.rb

```diff
@@ -156,6 +156,16 @@
     t.index ["user_id"], name: "index_projects_users_on_user_id"
   end
 
+  create_table "reports", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
```

Comment:
Wow what happened here...  I didn't expect to get this table back, sorry!    I think it was left over in my local DB.

Have to redo this.

Reply:
I think what happened here is that the old tables still exist on alpha.   So alpha doesn't have the latest code or somehow hasn't run the migrations that delete those tables.